### ISSUE TITLE
Code Explanation

### ISSUE THREAD

**Issue by tinkermachine** [CyberZHG/keras-bert#89]
Thank you for this contribution. Would appreciate your commentary on this code block in the classification demo:

**inputs = model.inputs[:2]
dense = model.get_layer('NSP-Dense').output
outputs = keras.layers.Dense(units=2, activation='softmax')(dense)**

Line 1: We are getting all layers EXCEPT last 2 layers  - OK got it.
**Line 2: Is there a layer named "NSP-Dense" that exists in the loaded model that you are extracting?**
Line 3: I think I understand the last line as you are adding a Dense layer for classification. OK here

**Comment by CyberZHG** [comment]
Line 2: The loaded model has fixed names. `NSP-Dense` is the first dense layer after the output of  `[CLS]` token.
